docs/api-routes/response-helpers.md

```diff
@@ -4,27 +4,99 @@ description: API Routes include a set of Express.js-like methods for the respons
 
 # Response Helpers
 
-<details open>
-  <summary><b>Examples</b></summary>
-  <ul>
-    <li><a href="https://github.com/vercel/next.js/tree/canary/examples/api-routes">Basic API Routes</a></li>
-    <li><a href="https://github.com/vercel/next.js/tree/canary/examples/api-routes-rest">API Routes with REST</a></li>
-  </ul>
-</details>
+The response (`res`) object includes a set of Express.js-like methods to improve the developer experience and increase the speed of creating new API endpoints.
 
-The response (`res`) includes a set of Express.js-like methods to improve the developer experience and increase the speed of creating new API endpoints, take a look at the following example:
+The included helpers are:
+
+- `res.status(code)` - A function to set the status code. `code` must be a valid [HTTP status code](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes)
+- `res.json(body)` - Sends a JSON response. `body` must be a [serialiazable object](https://developer.mozilla.org/en-US/docs/Glossary/Serialization)
+- `res.send(body)` - Sends the HTTP response. `body` can be a `string`, an `object` or a `Buffer`
+- `res.redirect([status,] path)` - Redirects to a specified path or URL. `status` must be a valid [HTTP status code](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes). If not specified, `status` defaults to "307" "Temporary redirect".
+
+## Setting the status code of a response
+
+When sending a response back to the client, you can set the status code of the response.
+
+The following example sets the status code of the response to `200` (`OK`) and returns a `name` property with the value of `John Doe` as a JSON response:
 
 ```js
 export default function handler(req, res) {
-  res.status(200).json({ name: 'Next.js' })
+  res.status(200).json({ name: 'John Doe' })
```

Comment:
Why "John Doe"? How about something like:

```suggestion
  res.status(200).json({ message: 'Hello from Next.js!' })
```

Reply:
Agreed 👍

Reply:
Totally. I grabbed John Doe from another example and went with it. Your suggestion is better though